### ISSUE TITLE
Create New Pull Request unknown error

### ISSUE THREAD

**Issue by ValeriyGourov** [MistyKuu/bitbucket-for-visual-studio#51]
A new error appears when creating a new pull request. However, in the future it does not interfere - the query is created normally.
![create new pull request unknown error](https://user-images.githubusercontent.com/24443483/36662224-5da0cc90-1ae6-11e8-8c7f-65b57c9d9801.png)

**Comment by MistyKuu** [comment]
Is it public or private bitbucket server? Please share logs located under %appdata%/gitclientvsextension/logs

**Comment by ValeriyGourov** [comment]
[logs.zip](https://github.com/MistyKuu/bitbucket-for-visual-studio/files/1768472/logs.zip)
As far as I know, this is a private bitbucket server.

**Comment by MistyKuu** [comment]
This is official bitbucket server. It cannot get default reviewers endpoint. It seems like you have to be repository admin.
```
2018-02-02 14:25:07,734 [1] INFO  BitBucket.REST.API.Wrappers.BitbucketRestClientBase [(null)] - Calling ExecuteTaskAsync. BaseUrl: https://api.bitbucket.org/2.0/ Resource: repositories/ukravit-team/ukravitapp/default-reviewers
2018-02-02 14:25:08,046 [94] ERROR BitBucket.REST.API.Serializers.NewtonsoftJsonSerializer [(null)] - Couldn't deserialize json: Forbidden. Error: Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: F. Path '', line 0, position 0.
   â Newtonsoft.Json.JsonTextReader.ParseValue()
   â Newtonsoft.Json.JsonTextReader.Read()
   â Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   â Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   â Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   â Newtonsoft.Json.JsonSerializer.Deserialize[T](JsonReader reader)
   â BitBucket.REST.API.Serializers.NewtonsoftJsonSerializer.Deserialize[T](String json)
2018-02-02 14:25:08,052 [1] ERROR GitClientVS.Infrastructure.Extensions.ViewModelsExtensions [(null)] - Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: F. Path '', line 0, position 0.
```

**Comment by ValeriyGourov** [comment]
I confused the server with the repository. I can not be a repository administrator because it's a company's repository. The reviewer is selected after entering several characters of the name. If there is no way to get any data, then at least correct the error correctly.

**Comment by AlanCarlyle** [comment]
I just got this error. One thing I noticed was my repository had space in it so git added single quotes around the name when cloning it. Could these quotes be the source of the problem? If they where serialised to json and json was set to use single quotes then this might cause a problem.
Haven't tested this theory yet myself.

**Comment by MistyKuu** [comment]
@AlanCarlyle Please put it in the browser:
https://api.bitbucket.org/2.0/repositories/{username}/{reponame}/default-reviewers
It's probably because you are not repo admin.

**Comment by MistyKuu** [comment]
Released in 1.37.2
